bin/compile

```diff
@@ -7,6 +7,21 @@ BUILD_DIR=$1
 CACHE_DIR=$2
 ENV_DIR=$3
 
+# SHA256 checksums of the sources hosted on GitHub at https://github.com/jemalloc/jemalloc/releases
+declare -A checksums=(
+  ['3.6.0']='e16c2159dd3c81ca2dc3b5c9ef0d43e1f2f45b04548f42db12e7c12d7bdf84fe'
+  ['4.0.4']='3fda8d8d7fcd041aa0bebbecd45c46b28873cf37bd36c56bf44961b36d0f42d0'
+  ['4.1.1']='c8f13b72b3aced936337a864fbe614a43e968f4adf64314b2a39c6ce43c9adae'
+  ['4.2.1']='5630650d5c1caab95d2f0898de4fe5ab8519dc680b04963b38bb425ef6a42d57'
+  ['4.3.1']='f7bb183ad8056941791e0f075b802e8ff10bd6e2d904e682f87c8f6a510c278b'
+  ['4.4.0']='a7aea63e9718d2f1adf81d87e3df3cb1b58deb86fc77bad5d702c4c59687b033'
+  ['4.5.0']='9409d85664b4f135b77518b0b118c549009dc10f6cba14557d170476611f6780'
+  ['5.0.1']='4814781d395b0ef093b21a08e8e6e0bd3dab8762f9935bbfb71679b0dea7c3e9'
+  ['5.1.0']='5396e61cc6103ac393136c309fae09e44d74743c86f90e266948c50f3dbb7268'
+  ['5.2.0']='74be9f44a60d2a99398e706baa921e4efde82bf8fd16e5c0643c375c5851e3b4'
+  ['5.2.1']='34330e5ce276099e2e8950d9335db5a875689a4c6a56751ef3b1d8c537f887f6'
+)
+
 # Default version
 version="5.2.1"
```

Comment:
Is it possible to override this somehow? Via Heroku env vars?
Maybe it would be possible to supply a checksum there, too?

Reply:
Yep, you can override it by setting `JEMALLOC_CHECKSUM`

https://github.com/gaffneyc/heroku-buildpack-jemalloc/pull/28/files#diff-a64a7d9b82578ca3053416d56631a61e0ceabed87619415663030e3842429e05R55